test/controllers/ChangeSuccessControllerSpec.scala

```diff
@@ -218,4 +224,54 @@ class ChangeSuccessControllerSpec extends ControllerBaseSpec with MockContactPre
       }
     }
   }
-}
+
+  "The .getVerifiedEmail function" when {
+
+    "the call to VatSubscription is successful" should {
+
+      "when the feature switch is on" should {
+
+        "return email verification status - emailVerified" in {
+          val result = {
+            mockIndividualAuthorised()
+            mockGetCustomerInfo(vrn)(Future.successful(Right(fullCustomerInfoModel)))
+            controller.getEmailVerifiedStatus(User(vrn)(request))
+          }
+          await(result) shouldBe Some(true)
+        }
+        "return email verification status - not verified" in {
+          val result = {
+            mockIndividualAuthorised()
+            mockGetCustomerInfo(vrn)(Future.successful(Right(minCustomerInfoModel)))
+            controller.getEmailVerifiedStatus(User(vrn)(request))
+          }
+          await(result) shouldBe None
+        }
+      }
+    }
+
+      "the feature switch is off" should {
+
+        "not call the .getVerifiedEmail function" in {
```

Comment:
```suggestion
        "not call the .getCustomerInfo function" in {
```
(this is the name of the vat subscription method it would call if feature switch was on)